package.json

```diff
@@ -7,7 +7,7 @@
     "npm": ">=5.0"
   },
   "dependencies": {
-    "babel-polyfill": "6.26.0",
+    "@babel/polyfill": "^7.2.5",
```

Comment:
Please don't use the `^` syntax in `dependencies`, only in `devDependencies`. Our bot will continuously try to update npm modules in package.json, but run them through PRs with CI, which is a lot more reliable.